### PR TITLE
Updates And Cleaning

### PR DESCRIPTION
**Pull request by ReyerBand**

### CODE REVIEW

**Comment by jsturdy** [comment]
If you can let me know when you're ready for this to be reviewed and merged, that would be great

**Review by cbravo135 — COMMENTED** [review]
We should just delete these files. I do not see us continuing to use them in the future. We have new updated versions that use a better data format.

File: macros/PLOTTING/plot_scurves_by_thresh.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+from plot_scurve import *
+from ROOT import TLegend
```
Comment (jsturdy) [inline macros/PLOTTING/plot_scurves_by_thresh.py:2]: ROOT imports need to happen after the option parsing

File: macros/PLOTTING/plot_noise_vs_trim.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from optparse import OptionParser
+from array import array
+from ROOT import TFile,TTree,TH1D,TGraph,TGraph2D,TCanvas,TPad,gROOT,gStyle,gPad,TPaveStats,TF1, TH2D
```
Comment (jsturdy) [inline macros/PLOTTING/plot_noise_vs_trim.py:3]: ROOT imports need to happen after the option parsing

File: macros/PLOTTING/plot_scurve.py [hunk 3]
```diff
@@ -46,6 +46,5 @@ def plot_scurve(VFAT, CH, fit_filename, overlay_fit, channel_yes):
             canvas.SaveAs('Scurve_VFAT%i_Strip%i.png'%(VFAT, CH))
             pass
         pass
-    return
+    return
```
Comment (jsturdy) [inline macros/PLOTTING/plot_scurve.py:49]: Trailing whitespace should be removed/avoided (```emacs``` command ```M-x delete-trailing-whitespace```)

File: macros/PLOTTING/plot_vfat_summary.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
+from optparse import OptionParser
+from array import array
+from ROOT import TFile,TTree,TH1D,TGraph,TGraph2D,TCanvas,TPad,gROOT,gStyle,gPad,TPaveStats,TF1, TH2D
```
Comment (jsturdy) [inline macros/PLOTTING/plot_vfat_summary.py:3]: ROOT imports need to happen after the option parsing

File: macros/anaUltraScurve.py [hunk 5]
```diff
@@ -3,7 +3,8 @@
 from array import array
 from fitScanData import *
 from channelMaps import *
-from ROOT import TFile,TTree,TH2D,TGraph,TGraph2D,TCanvas,TPad,gROOT,gStyle,gPad,TPaveStats
+from PanChannelMaps import *
+from ROOT import TFile,TTree,TH2D,TGraph,TGraph2D,TCanvas,TPad,gROOT,gStyle,gPad,TPaveStats, TLine, TColor, TLegend
```
Comment (jsturdy) [inline macros/anaUltraScurve.py:7]: ROOT imports need to happen after the option parsing

File: macros/anaUltraScurve.py [hunk 6]
```diff
@@ -204,15 +247,47 @@ def overlay_fit(VFAT, CH):
...
+            legend.Draw('SAME')
+            print trimVcal[i]
+            lines[i].SetLineColor(1)
+            lines[i].SetLineWidth(3)
+            lines[i].Draw('SAME')
+            pass
+        canv.Update()
+        pass
     pass
+<<<<<<< HEAD
```
Comment (jsturdy) [inline macros/anaUltraScurve.py:270]: Missed a conflict resolution

File: macros/anaUltraScurve.py [hunk 7]
```diff
@@ -204,15 +247,47 @@ def overlay_fit(VFAT, CH):
...
+            canv.Update()
+            canv.cd((i+9 + j*16)%48 + 16)
+            vSum2[i+(8*j)].Draw('colz')
+            canv.Update()
+            pass
+        pass
+    pass
+
+canv.SaveAs(filename+'/Summary.png')
+>>>>>>> glib-only-fw
```
Comment (jsturdy) [inline macros/anaUltraScurve.py:290]: ended here

**Review by cbravo135 — APPROVED** [review]
Yay, more analysis tools!

**Review by jsturdy — APPROVED** [review]
I will fix the issues mentioned in my review

**Pull request merged**